ledger-api/grpc-definitions/com/daml/ledger/api/v1/commands.proto

```diff
@@ -217,9 +218,17 @@ message DisclosedContract {
   // The contract id
   // Required
   string contract_id = 2;
+
   // The contract arguments
-  // Required
-  Record arguments = 3;
+  oneof arguments {
+    // The contract arguments as typed Record
+    Record record = 3;
+
+    // Contract argument accessible at ``com.daml.ledger.api.v1.CreatedEvent.create_arguments_blob``
+    // should be specified here.
```

Comment:
@skisel-da : I don't understand this comment. Can you clarify?

Reply:
The blob which is expected here is the one which is exposed via `CreatedEvent.create_arguments_blob`, this is to link these blobs together.